proj_eclipse/src/main/java/smartmag/models/BoxModel.java

```diff
@@ -201,7 +202,7 @@ protected void preleva(int qta) {
 	 * 
 	 * @param qta
 	 */
-	private void setQuantita(int qta) {
+	public void setQuantita(int qta) {
```

Comment:
Solo i modelli (MovimenModel) dovrebbe modificarla
```suggestion
	protected void setQuantita(int qta) {
```